ui/pages/Search/selectors.js

```diff
@@ -192,3 +247,37 @@ export const getAnalysisGroupOptions = createSelector(
     ...(analysisGroupsGroupedByProjectGuid.null || {}),
   }).map(group => ({ value: group.analysisGroupGuid, text: group.name, icon: group.criteria ? 'sync' : null })),
 )
+
+export const getDisplayVariants = createSelector(
+  getFlattenCompoundHet,
```

Comment:
this si actually a small code change- previously in the search page we were mapping state to props with `getFlattenCompoundHet`, passing it around as a prop, and then when this selector was used it used the prop. This simplifies that flow by just using the state directly